src/demo-app/radio/radio-demo.html

```diff
@@ -28,3 +28,30 @@
   </md-radio-group>
   <p>Your favorite season is: {{favoriteSeason}}</p>
 </section>
+<h1>ARIA Example</h1>
```

Comment:
Not sure about adding an extra demo, just for the `aria-label` and `aria-labelled-by` attribute. It seems unnecessary to me.